.scripts/pre-push

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+set -o nounset
+set -o errexit
+
+# If the publication-model has updates that change the model structure,
+# the model version should be updated.
+
+TARGET="publication-model"
+BUILD_FILE="${TARGET}/build.gradle"
+PATTERN="^def modelVersion = '"
+MODEL_CHANGE_MESSAGE="Changes in the $TARGET package, which imply\
+ the model version number may need updating. If there are no effective model structure changes,\
+ override this pre-push hook with $ git push --no-verify"
+NO_MODEL_CHANGE_MESSAGE="No changes to $TARGET. Continuing."
```

Comment:
Receiving this message also when the model version is updated. Maybe misleading?